xarray/core/dataset.py

```diff
@@ -4165,6 +4165,20 @@ def apply(
         attrs = self.attrs if keep_attrs else None
         return type(self)(variables, attrs=attrs)
 
+    def apply(
+        self,
+        func: Callable,
+        keep_attrs: bool = None,
+        args: Iterable[Any] = (),
+        **kwargs: Any,
+    ) -> "Dataset":
```

Comment:
Can we keep the docstring here, even if it's just "Alias for Dataset.map"?

Ideally we would use the `See also` section, which gets turned into a link by numpydoc.

Reply:
Yes for sure, that was an oversight (e.g. I kept them as references for `drop`)

Reply:
Also added `See Also` for the drop cases

Reply:
Yes for sure